src/install-ocaml-windows.cmd

```diff
@@ -1,5 +1,6 @@
 set CYGWIN_ROOT=c:\cygwin
 %2/setup-x86_64.exe --quiet-mode --root %CYGWIN_ROOT% --site http://cygwin.mirror.constant.com --packages curl,diff,diffutils,git,m4,make,patch,perl,rsync,mingw64-x86_64-gcc-core,unzip
+copy %2/setup-x86_64.exe %CYGWIN_ROOT%
```

Comment:
You've updated `src/install-ocaml-windows.cmd` but not `lib/install-ocaml-windows.cmd` (which you can either do manually, or run the appropriate `yarn` command).

When you do, though, this command will fail because of the `/`. This line _requires_ `%2\` - the command above would be more resilient if it were `%s\setup-x86_64.exe`.

Reply:
> You've updated `src/install-ocaml-windows.cmd` but not `lib/install-ocaml-windows.cmd` (which you can either do manually, or run the appropriate `yarn` command).

Will do...
 
> When you do, though, this command will fail because of the `/`. This line _requires_ `%2\` 

True... "copy" would see that as a switch. I'll fix it.

> - the command above would be more resilient if it were `%s\setup-x86_64.exe`.

I'm just using what's already in the CMD file. I'm reluctant to use a separate path, maybe that should be changed for all the respective instances?

Reply:
Fix commit pushed ... (with plan to squash when appropriate).